pkg/core/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -396,6 +396,9 @@ one = 'unable to create new Kafka instance at this time in specified cloud provi
 [kafka.create.error.instance.limit]
 one = 'maximum number of allowed kafka instances has been reached. Please review all instances that your user has access to and delete one or more instances before creating a new one'
```

Comment:
Suggest: 'the maximum number of Kafka instances for your organization has been reached. Before you can create a new instance, you must delete one or more of the existing instances'